src/polyfills/fetchWithTimeout.js

```diff
@@ -0,0 +1,37 @@
+// If the server takes more than 2 second to respond; throw and Error as Request timed out
```

Comment:
@ManujJiraUser,
Please remove the commented out code.

Reply:
@mandy8055,
The changes have been factored in. So resolving this comment.